templates/dockstore.model.ts.template

```diff
@@ -3,14 +3,14 @@ export class Dockstore {
 
   // Please fill in HOSTNAME with your address
   static readonly HOSTNAME = 'http{{#HTTPS}}s{{/HTTPS}}://{{ DOMAIN_NAME }}';
-  static readonly API_PORT = '8443';
+  static readonly API_PORT = '443';
   static readonly UI_PORT = '443';
 
   // Discourse URL MUST end with a slash (/)
   static readonly DISCOURSE_URL = '{{ DISCOURSE_URL }}';
 
   static readonly LOCAL_URI = Dockstore.HOSTNAME + ':' + Dockstore.UI_PORT;
-  static readonly API_URI = Dockstore.HOSTNAME + ':' + Dockstore.API_PORT;
+  static readonly API_URI = Dockstore.HOSTNAME + ':' + Dockstore.API_PORT + '/api';
```

Comment:
Does this risk breaking existing clients, in particular FireCloud (and maybe DNAstack)? Can we have 8443 redirect to 443? Maybe you already do this, as I don't know nginx config very well, but I don't think I see it.

Reply:
This is the UI, so the UI will always point at https://staging.dockstore.org/api/swagger.json style endpoints now. But the nginx will reverse proxy calls to both now which is why https://staging.dockstore.org:8443/metadata/dockerRegistryList and https://staging.dockstore.org/api/metadata/dockerRegistryList both work without 301 redirects.